commands/commands.go

```diff
@@ -14,23 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-/* Zeitgeist is a language-agnostic dependency checker */
-
-package cmd
+package commands
 
 import (
-	"github.com/sirupsen/logrus"
+	"fmt"
+
 	"github.com/spf13/cobra"
 
 	"sigs.k8s.io/release-utils/log"
 )
 
-type rootOptions struct {
-	logLevel string
-}
+const defaultConfigFile = "dependencies.yaml"
 
 var (
-	rootOpts = &rootOptions{}
+	rootOpts = &options{}
```

Comment:
are we saying that we want all options to be global to all sub-commands? how would we handle any options that's specific to a sub-command?